### PR TITLE
BookClub model created

### PR DESCRIPTION
**Pull request by larisagranat**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,12 +10,19 @@
...
-ActiveRecord::Schema.define(version: 2019_08_15_015807) do
+ActiveRecord::Schema.define(version: 2019_08_19_020321) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "hstore"
   enable_extension "plpgsql"
 
+  create_table "admins_book_clubs", force: :cascade do |t|
+    t.bigint "admins_id"
+    t.bigint "book_club_id"
```
Comment (YDick) [inline db/schema.rb]: why bigint?

**Comment by YDick** [comment]
When I tried to migrate I got:
SyntaxError: /home/yehudis/Desktop/adalegIT/GroupProj/BookIt/db/migrate/20190815015807_create_book_clubs_users.rb:6: syntax error, unexpected end-of-input, expecting keyword_end

There was a missing end, so I added it and the migration went through. Im not sure if it will do what you want it too now though.

File: db/migrate/20190818163655_rename_table.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class RenameTable < ActiveRecord::Migration[5.2]
+  def change
+
+    def self.up
+      rename_table :admins_book_clubs, book_clubs_admins
```
Comment (YDick) [inline db/migrate/20190818163655_rename_table.rb]: There is a typo: 
rename_table :admins_book_clubs, book_clubs_admins
should be:
rename_table :admins_book_clubs, **:**book_clubs_admins

**Comment by YDick** [comment]
Also, in your terminal, do a "git fetch origin master" and a "git merge" occasionally to keep up with updates in the master branch in github. I added a gitignore schema in the master branch. It was too confusing with the migrations. Also, try not to delete migrations

**Comment by larisagranat** [comment]
will push updated BookClub model

**Pull request closed without merging**